Library/Homebrew/extend/os/mac/software_spec.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class SoftwareSpec
+  undef uses_from_macos
```

Comment:
Not sure about this, i was troubleshooting a misterious spec that was failling because it requires linux and it wasn't mocking with `OS.linux?`. Saw this on other files and added here...